src/store/console/mutations.ts

```diff
@@ -114,5 +114,9 @@ export const mutations: MutationTree<ConsoleState> = {
       state.consoleFilters.splice(i, 1)
       state.consoleFiltersRegexp.splice(i, 1)
     }
+  },
+
+  setLastCleared (state) {
+    Vue.set(state, 'lastCleared', Date.now())
```

Comment:
Ideally, we would use the latest received server timestamp here instead of `Date.now()` but for now this seems to do the trick just fine!